sourcegraph/sourcegraph.proto

```diff
@@ -1850,7 +1850,7 @@ service Auth {
 	// authorization levels on the client.
 	rpc GetPermissions(pbtypes.Void) returns (UserPermissions) {
 		option (google.api.http) = {
-			get: "/auth/verify"
+			get: "/auth/getpermissions"
```

Comment:
All other endpoints currently use underscore separators between words (e.g., see [here](https://github.com/sourcegraph/go-sourcegraph/blob/4a14f24f8eaeae876efb23cc9647db6fd5744764/sourcegraph/sourcegraph.proto#L463-L509), so for consistency, this should be:

```
get: "/auth/get_permissions"
```

Otherwise this is good.